BUILD.gn

```diff
@@ -0,0 +1,507 @@
+    "runtime/browser/android/xwalk_autofill_client_android.cc",
+    "runtime/browser/android/xwalk_autofill_client_android.h",
+    "runtime/browser/android/xwalk_content.cc",
+    "runtime/browser/android/xwalk_content.h",
+    "runtime/browser/android/xwalk_content_lifecycle_notifier.cc",
+    "runtime/browser/android/xwalk_content_lifecycle_notifier.h",
+    "runtime/browser/android/xwalk_contents_client_bridge.cc",
+    "runtime/browser/android/xwalk_contents_client_bridge.h",
+    "runtime/browser/android/xwalk_contents_client_bridge_base.cc",
+    "runtime/browser/android/xwalk_contents_client_bridge_base.h",
+    "runtime/browser/android/xwalk_contents_io_thread_client.h",
+    "runtime/browser/android/xwalk_contents_io_thread_client_impl.cc",
+    "runtime/browser/android/xwalk_contents_io_thread_client_impl.h",
+    "runtime/browser/android/xwalk_cookie_access_policy.cc",
+    "runtime/browser/android/xwalk_cookie_access_policy.h",
+    "runtime/browser/android/xwalk_dev_tools_server.cc",
+    "runtime/browser/android/xwalk_dev_tools_server.h",
+    "runtime/browser/android/xwalk_download_resource_throttle.cc",
+    "runtime/browser/android/xwalk_download_resource_throttle.h",
+    "runtime/browser/android/xwalk_http_auth_handler.cc",
+    "runtime/browser/android/xwalk_http_auth_handler.h",
+    "runtime/browser/android/xwalk_http_auth_handler_base.cc",
+    "runtime/browser/android/xwalk_http_auth_handler_base.h",
+    "runtime/browser/android/xwalk_icon_helper.cc",
+    "runtime/browser/android/xwalk_icon_helper.h",
+    "runtime/browser/android/xwalk_login_delegate.cc",
+    "runtime/browser/android/xwalk_login_delegate.h",
+    "runtime/browser/android/xwalk_path_helper.cc",
+    "runtime/browser/android/xwalk_path_helper.h",
+    "runtime/browser/android/xwalk_presentation_host.cc",
+    "runtime/browser/android/xwalk_presentation_host.h",
+    "runtime/browser/android/xwalk_request_interceptor.cc",
+    "runtime/browser/android/xwalk_request_interceptor.h",
+    "runtime/browser/android/xwalk_settings.cc",
+    "runtime/browser/android/xwalk_view_delegate.cc",
+    "runtime/browser/android/xwalk_view_delegate.h",
+    "runtime/browser/android/xwalk_web_contents_delegate.cc",
+    "runtime/browser/android/xwalk_web_contents_delegate.h",
+    "runtime/browser/android/xwalk_web_contents_view_delegate.cc",
+    "runtime/browser/android/xwalk_web_contents_view_delegate.h",
+    "runtime/browser/android/xwalk_web_resource_response.cc",
+    "runtime/browser/android/xwalk_web_resource_response.h",
+    "runtime/browser/android/xwalk_web_resource_response_impl.cc",
+    "runtime/browser/android/xwalk_web_resource_response_impl.h",
+    "runtime/browser/application_component.cc",
+    "runtime/browser/application_component.h",
+    "runtime/browser/devtools/remote_debugging_server.cc",
+    "runtime/browser/devtools/remote_debugging_server.h",
+    "runtime/browser/devtools/xwalk_devtools_frontend.cc",
```

Comment:
`xwalk.gyp` needs to manually remove a bunch of files from the list depending on whether `OS=="android"`. Since the GN way is to actually _add_ stuff to the list based on variables, I think you need to create an `if (!is_android)` block below and put these entries there.